<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMain</class>
 <widget class="QMainWindow" name="WMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>684</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Arkanoid Demo</string>
  </property>
  <widget class="QWidget" name="wd_central">
   <widget class="GameFrame" name="fr_game">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="GameState" name="fr_state">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>291</width>
      <height>617</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_GameName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;Arkanoid Solid&quot;;
color: rgb(7, 120, 253);
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>ARKANOID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lt_Score1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_ScoreCaption">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Practice-font&quot;;
color: rgb(255, 160, 0);
color: rgb(255, 0, 0);
</string>
         </property>
         <property name="text">
          <string>SCORE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Device_Background" name="db_Score" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_Score">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Orbit-B BT&quot;;
color :  rgb(2, 250, 3);
</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lt_HighScore">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_HighScoreCaption">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Practice-font&quot;;
color: rgb(255, 160, 0);
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>HIGH SCORE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Device_Background" name="db_HighScore" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_HighScore">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Orbit-B BT&quot;;
color :  rgb(2, 250, 3);
</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lt_Level">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_LevelCaption">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Practice-font&quot;;
color: rgb(255, 160, 0);
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>LEVEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Device_Background" name="db_Level" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_Level">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Orbit-B BT&quot;;
color :  rgb(2, 250, 3);
</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lt_Lifes">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_LifesCaption">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Practice-font&quot;;
color: rgb(255, 160, 0);
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>LIFES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Device_Background" name="db_Lifes" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_Lifes">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Orbit-B BT&quot;;
color :  rgb(2, 250, 3);
</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lt_ActivePrizes">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_ActivePrizesCaption">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Practice-font&quot;;
color: rgb(255, 160, 0);
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>ACTIVE PRIZES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Device_Prizes_List" name="dpl_ActivePrizes" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>debug 0 level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenu {
	background-color: rgb(0, 0, 0);
	color :  rgb(2, 250, 3);
   font-weight: bold;
  border-style: solid;
 border-width: 2px;
 border-color:  rgb(2, 250, 3);

}</string>
    </property>
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionWins_Statistic"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu {
	background-color: rgb(0, 0, 0);
	color :  rgb(2, 250, 3);
   font-weight: bold;

border-style: solid;
 border-width: 2px;
 border-color:  rgb(2, 250, 3);
}</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="styleSheet">
     <string notr="true">QMenu {
	background-color: rgb(0, 0, 0);
	color :  rgb(2, 250, 3);
   font-weight: bold;

border-style: solid;
 border-width: 2px;
 border-color:  rgb(2, 250, 3);


}</string>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionLocale"/>
    <addaction name="actionSettings_Window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLocale">
   <property name="text">
    <string>Locale</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings_Window">
   <property name="text">
    <string>Settings Window</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionWins_Statistic">
   <property name="text">
    <string>Wins Statistic</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameFrame</class>
   <extends>QFrame</extends>
   <header>gameframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Device_Background</class>
   <extends>QWidget</extends>
   <header>device_background.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameState</class>
   <extends>QFrame</extends>
   <header>gamestate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Device_Prizes_List</class>
   <extends>QWidget</extends>
   <header>device_prizes_list.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
